code/game/objects/items/weapons/manuals.dm

```diff
@@ -601,4 +601,309 @@
                 </ol>
 
 				</body>
-				</html>"}
\ No newline at end of file
+				</html>"}
+
+/obj/item/weapon/book/manual/virology_guide
+	name = "A Crash Course in Virology"
+	icon_state ="bookVirologyGuide"
+	author = "Frederick Chapman Montagnier"
+	title = "A Crash Course in Virology"
+	book_width = 819
+	book_height = 516
+	dat = {"<html>
+				<head>
+				<style>
+				h1 {
+				  font-family: Arial, Helvetica, sans-serif;
+				  text-align:center;
+				  margin-bottom: 0;
+				}
+				h2 {
+				  font-family: Arial, Helvetica, sans-serif;
+				  color: grey;
+				  text-align:center;
+				  margin-top: 0;
+				}
+				h3 {
+				  font-family: Arial, Helvetica, sans-serif;
+				}
+				p,li {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				}
+				dt {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				  font-weight: bold;
+				}
+				dd {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				}
+				table {
+				  font-family: Arial, Helvetica, sans-serif;
+				  border-collapse: collapse;
+				  width:100%;
+				  border: 2px solid black;
+				}
+
+				td, th {
+				  border: 1px solid gray;
+				}
+				</style>
+				</head>
+				<body>
+				<h1>A Crash Course in Virology</h1>
+				<h2>and pathogenics in general</h2>
+				<ul id="menu">
+				  <li><a href="#chapter0">Preamble: HOW DO I CURE VIRUS</a></li>
+				  <li><a href="#chapter1">Chapter 1: Proper Safety</a></li>
+				  <li><a href="#chapter2">Chapter 2: Acquiring Pathogenic Samples</a></li>
+				  <li><a href="#chapter3">Chapter 3: Incubating and Analyzing Samples</a></li>
+				  <li><a href="#chapter4">Chapter 4: Pathogen Modification</a></li>
+				  <li><a href="#chapter5">Chapter 5: Overview of the Various Cures and the Immune System</a></li>
+				  <li><a href="#chapter6">Afterword: On your Purpose as Virologist</a></li>
+				</ul>
+				<h3 id="chapter0">Preamble: Quick, how do I cure a disease?</h3>
+				<p>Because lets be honest, unless you are a diplomed Pathologist, this is most likely the reason why you openned this manual. Now keep calm, put on a sterile mask and some latex gloves, and follow these steps:</p>
```

Comment:
```suggestion
				<p>Because let's be honest, unless you are a qualified Pathologist, this is most likely the reason why you opened this manual. Now keep calm, put on a sterile mask and some latex gloves, and follow these steps:</p>
```